src/main/java/com/batix/rundeck/plugins/AnsibleResourceModelSource.java

```diff
@@ -290,12 +294,12 @@ public INodeSet getNodes() throws ResourceModelSourceException {
           // ansible_system     = Linux   = osFamily in Rundeck
           // ansible_os_family  = Debian  = osName in Rundeck
 
-          if (root.has("ansible_system")) {
-            node.setOsFamily(root.get("ansible_system").getAsString());
+          if (root.has("ansible_os_family")) {

```

Comment:
Hi, this worked for me on ansible 2.3.1.0.

<img width="717" alt="screenshot 2017-08-09 22 39 52" src="https://user-images.githubusercontent.com/6034968/29152145-b84ed582-7d53-11e7-8117-f4f5afd042e4.png">

<img width="1252" alt="screenshot 2017-08-09 22 44 34" src="https://user-images.githubusercontent.com/6034968/29152236-59822e9a-7d54-11e7-810e-baefcb8a83f9.png">